scripts/RNA_analysis/make_complexity_plots.py

```diff
@@ -46,11 +48,19 @@
 #SBATCH {5}
 
 module load bioinfo-tools
+module load {6}
 module load {3}
 module load {2}
 
 cd {4}
+""".format(name, mail, preseq, bam, path, extra_arg, bed)
+
+if len(sys.argv)==6:
+	print >>f, """bedtools bamtobed -i tophat_out_{0}/accepted_hits_sorted_{0}.bam | sort -k 1,1 -k 2,2n -k 6,6 >tophat_out_{0}/accepted_hits_sorted_preseq_{0}.bed
```

Comment:
Preseq complained about the "maximum segment length" if the 'TLEN' value <0
in BAM file. It seems there's a problem to parse the strand info. in BAM.

On Mon, Sep 21, 2015 at 11:28 AM, Phil Ewels <notifications@github.com>
wrote:

> In scripts/RNA_analysis/make_complexity_plots.py
> <https://github.com/SciLifeLab/scilifelab/pull/484#discussion_r39953290>:
>
> >  module load {3}
> >  module load {2}
> >
> >  cd {4}
> > +""".format(name, mail, preseq, bam, path, extra_arg, bed)
> > +
> > +if len(sys.argv)==6:
> > +	print >>f, """bedtools bamtobed -i tophat_out_{0}/accepted_hits_sorted_{0}.bam | sort -k 1,1 -k 2,2n -k 6,6 >tophat_out_{0}/accepted_hits_sorted_preseq_{0}.bed
>
> Interesting! What happened?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/scilifelab/pull/484/files#r39953290>.
>